wren-ai-service/README.md

```diff
@@ -66,11 +66,6 @@ The following commands can quickly start the service for development:
   - wren-ui: port should be 3000
   - qdrant: ports should be 6333, 6334
 
-## Adding your preferred LLM, Embedder or Document Store
+## Contributing
 
-Please read the [documentation](https://docs.getwren.ai/installation/custom_llm) here to check out how you can add your preferred LLM, Embedder or Document Store.
-
-## Related Issues or PRs
-
-- [Issues](https://github.com/Canner/WrenAI/issues?q=is%3Aopen+is%3Aissue+label%3Amodule%2Fai-service)
-- [PRs](https://github.com/Canner/WrenAI/pulls?q=is%3Aopen+is%3Apr+label%3Amodule%2Fai-service)
+Thank you for investing your time in contributing to our project! Please [read this for more information](CONTRIBUTING.md)!
```

Comment:
I'm thinking about moving the section before the environment setup. If I were a newcomer and found the contributing guide, I might find it confusing if the first step sends me back to a previous page to set up the development environment lol. What do you think?

Reply:
I think it's ok. Since it's hard for users to easily contribute the project. Oftentimes, when they really want to contribute the project, they've seen the requirements. So the requirements are reminder in my opinion.

Reply:
okay. let's keep it there.